bin/main

```diff
@@ -1,66 +1,123 @@
 #!/usr/bin/env ruby
 
-# select game option method
-puts 'Welcome to Tic-Tac-Toe game! Please select your option'
-puts '1 - Play'
-puts '2 - Exit'
-option = gets.chomp.to_i
-puts option
-# select option method ends here
-
-if option == 2
-  puts 'Thank you! See you soon!!'
-elsif option == 1
-  puts 'Welcome!! Your match will start soon..'
-  # will call game start method here
-else
-  puts 'Invalid option. Please enter the correct option.'
-  # will call the game option method
+class Player
+  attr_accessor :name, :key, :array
+  def initialize(name, key)
+    @name = name
+    @key = key
+    @array = []
+  end
 end
 
-# call game start method
-2.times do
-  # On each iteration an instance of class player will be created
-  # The players will be automatically assigned the keys
-  # X for player and O for player2
-  puts 'Enter your name'
-  name = gets.chomp
-  puts name
-end
+class Game
+  WINNING_COMBO = [
+    [1, 2, 3],
+    [4, 5, 6],
+    [7, 8, 9],
+    [1, 4, 7],
+    [2, 5, 8],
+    [3, 6, 9],
+    [1, 5, 9],
+    [3, 5, 7]
+  ].freeze
+  attr_accessor :player1, :player2, :game_array
```

Comment:
`attr_reader` seems to be more appropriate because there are only used as reader in your code.
Kindly try to not use `attr_accessor` as indicated [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png)